src/components/Feed/FeedModeSelector.tsx

```diff
@@ -0,0 +1,61 @@
+import { MouseEventHandler, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { useTrack } from '~/contexts/analytics/AnalyticsContext';
+
+import TextButton, { StyledButtonText } from '../core/Button/TextButton';
+import colors from '../core/colors';
+import { HStack } from '../core/Spacer/Stack';
+import { FeedMode } from './Feed';
+
+type Props = {
+  feedMode: FeedMode;
+  setFeedMode: (mode: FeedMode) => void;
+};
+
+export default function FeedModeSelector({ feedMode, setFeedMode }: Props) {
```

Comment:
mostly existing logic that I moved from the old FeedNavbar into this file

logic and structure is similar, just without using the Navbar components